src/main/java/edu/cornell/kfs/rass/batch/service/RassReportService.java

```diff
@@ -4,7 +4,7 @@
 
 public interface RassReportService {
     
-    void writeBatchJobReport(RassBatchJobReport rassBatchJobReport);
+    void writeBatchJobReports(RassBatchJobReport rassBatchJobReport);
     
     void sendReportEmail();
```

Comment:
since it looks like sendReportEmail() is just used internally to RassReportServiceImpl, should this function no longer be exposed on the interface?

Reply:
Good point, I'll remove it from the interface.